.github/workflows/gh_pages.yml

```diff
@@ -1,30 +1,30 @@
 name: Github Pages
 on:
-  release:
-    types: [published]
+  push:
+    branches:
+      - master
```

Comment:
❗  With the old trigger (release published) the gh-pages would be updated only when we launch a release. With this change though it will be updated every time a new code is merged into master. Is that your intention?

Nowadays you can re-run this script simply by converting a release to draft and then re-publish it again.

Reply:
That was the intent, yeah, so that we could iterate on the file without publishing a release (since that would trigger the whole `npm_publish` routine and is generally more involved).